amlip_docs/index.rst

```diff
@@ -58,6 +59,7 @@
 
    Scenarios </rst/user_manual/scenarios/scenarios>
```

Comment:
I think this is what we have in the demos, not sure how to do it without replicating it.

Reply:
The one in the demos is a Demo. I meant here to add a "Scenario" as a context where Inference could run. It would be to explain in a single file the nodes and the data types all related. Check other scenarios

Reply:
not apply